dotnet/src/Experimental/Process.LocalRuntime/LocalProcess.cs

```diff
@@ -149,7 +149,7 @@ internal override async Task HandleMessageAsync(ProcessMessage message)
         {
             // Create the external event that will be used to start the nested process. Since this event came
             // from outside this processes, we set the visibility to internal so that it's not emitted back out again.
-            var nestedEvent = new KernelProcessEvent() { Id = eventId, Data = message.TargetEventData, Visibility = KernelProcessEventVisibility.Internal };
+            KernelProcessEvent nestedEvent = new() { Id = eventId, Data = message.TargetEventData };
```

Comment:
Curious about losing the visibility change here. If this is correct, then I think the comment needs to be updated.

Reply:
Ah right, will restore.